File: C++/734A. Anton and Danik.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n, a=0, d=0;
    string s;
    cin>>n;
    cin>>s;
    for(int i=0; i<n; i++){
        if (s[i] == 'A'){
            a++;
        }
        else{
            d++;
        }
    }
    if(a>d){
        cout<<"Anton";
    }
    else if(d>a){
        cout<<"Danik";
    }
    else{
        cout<<"Friendship";
    }
    cout<<endl;
    return 0;
}

File: Python/734A. Anton and Danik.py
n =  int(input())
a, d = 0, 0
s = input()
for i in range(n):
    if s[i] == 'A':
        a += 1
    else:
        d += 1
if a>d:
    print("Anton\n")
elif a<d:
    print("Danik\n")
else:
    print("Friendship\n")